--- a/Package.pm	2016-04-25 23:41:39.000000000 +0200
+++ b/Package.pm	2016-08-18 11:34:34.281978936 +0200
@@ -70,7 +70,7 @@
 (?:^|/)(?:DEADJOE|\.arch-inventory|\.(?:bzr|cvs|hg|git)ignore)$|
 # File or directory names that should be ignored
 (?:^|/)(?:CVS|RCS|\.deps|\{arch\}|\.arch-ids|\.svn|
-\.hg(?:tags|sigs)?|_darcs|\.git(?:attributes|modules)?|
+\.hg(?:tags|sigs)?|_darcs|\.git(?:attributes|modules|review)?|
 \.shelf|_MTN|\.be|\.bzr(?:\.backup|tags)?)(?:$|/.*$)
 ';
 # Take out comments and newlines
@@ -106,6 +106,7 @@
 .gitattributes
 .gitignore
 .gitmodules
+.gitreview
 .hg
 .hgignore
 .hgsigs
